src/examples/poc_noresp_ping.nim

```diff
@@ -105,26 +107,11 @@ proc mixnetSimulation() {.async.} =
   if senderIndex < numberOfNodes - 1:
     receiverIndex = senderIndex + 1
 
-  var sendDialerFunc = proc(
-      msg: seq[byte],
-      proto: ProtocolType,
-      destMultiAddr: MultiAddress,
-      destPeerId: PeerId,
-  ): Future[void] {.async: (raises: [CancelledError, LPStreamError]).} =
-    try:
-      await mixProto[senderIndex].anonymizeLocalProtocolSend(
-        msg, proto, destMultiAddr, destPeerId
-      )
-    except CatchableError as e:
-      error "Error during execution of sendThroughMixnet: ", err = e.msg
-      # TODO: handle error
-    return
-
-  let conn = MixEntryConnection.new(
-    nodes[receiverIndex].peerInfo.addrs[0],
+  let conn = createMixEntryConnection(
+    mixProto[senderIndex],
+    some(nodes[receiverIndex].peerInfo.addrs[0]),
     nodes[receiverIndex].peerInfo.peerId,
-    ProtocolType.fromString(NoRespPingCodec),
-    sendDialerFunc,
```

Comment:
looks like this is the answer to the question posed here: https://github.com/vacp2p/mix/pull/40#discussion_r2164810555